app/src/main/java/com/gholem/moneylab/features/chooseTransactionCategory/viewmodel/BottomSheetCategoryViewModel.kt

```diff
@@ -21,11 +21,11 @@ class BottomSheetCategoryViewModel @Inject constructor(
 
     private val _actions = Channel<Action>(Channel.BUFFERED)
     val actions = _actions.receiveAsFlow()
-    var listOfCategories = mutableListOf<TransactionCategoryModel>()
+    val listOfCategories = mutableListOf<TransactionCategoryModel>()
     val navigation: NavigationLiveData<BottomSheetCategoryEvent> = NavigationLiveData()
 
     fun getCategories() = viewModelScope.launch {
-        listOfCategories = getCategoryListUseCase.run(Unit) as MutableList<TransactionCategoryModel>
+        listOfCategories.addAll(getCategoryListUseCase.run(Unit) as MutableList<TransactionCategoryModel>)
```

Comment:
If you do that this way then you have to clear `listOfCategories` first - otherwise you can end up with adding next set of categories to the existing one and so on.